tfexec/terraform_test.go

```diff
@@ -19,11 +20,30 @@ const testConfigFileName = "main.tf"
 const testStateJsonFileName = "state.json"
 const testTerraformStateFileName = "terraform.tfstate"
 
+var tfPath string
+
+func TestMain(m *testing.M) {
+	var err error
+	td, err := ioutil.TempDir("", "tfinstall")
+	if err != nil {
+		panic(err)
+	}
+	defer os.RemoveAll(td)
+
+	tfPath, err = tfinstall.Find(tfinstall.LookPath(), tfinstall.LatestVersion(td, true))
```

Comment:
You may want to add a finder here that takes an exact version from an env var if specified, so we can more easily matrix test this like `terraform-plugin-test`

Reply:
Thanks, this is a good suggestion for #19 - we won't need it until then.